packages/playht/src/grpc-client/client.ts

```diff
@@ -138,7 +138,7 @@ export class Client {
       this.leasePromise = undefined;
     }
 
-    const address = this.options.customAddr ?? this.lease.metadata.inference_address;
+    const address = this.lease.metadata.inference_address;
```

Comment:
This code was just plain wrong.

- `this.rpc` should use the regular inference address returned by the lease
- `this.premiumRpc` should use the premium inference address returned by the lease
- `this.customRpc` should use the custom address (if any) specified by the customer

Then, when we construct the tts stream source, we pass the appropriate rpc clients.

Before this change, the tts stream source would get TWO rpc clients that point to the custom-specified custom address -- so fallback would fall back to the custom address!